constructor/osxpkg.py

```diff
@@ -50,6 +50,8 @@ def write_readme(dst, info):
                 continue
             f.write("{\\listtext\t\n\\f1 \\uc0\\u8259 \n\\f0 \t}%s %s\\\n" %
                     tuple(dist.rsplit('-', 2)[:2]))
+        f.write(f"\n\\pard\\pardeftab720\\partightenfactor0\n\\cf0 \\\nCreated by "
+                f"constructor {info['CONSTRUCTOR_VERSION']}\\\n")
```

Comment:
looked into that already for a while and tried some things:
* it is possible to see the current attributes with `mdls`, e.g. `mdls Miniconda3-py310_22.9.0-rc0-18-g2ddab6d-MacOSX-arm64.pkg
* the attributes I was able to change are either stored in the spotlight db or via via extended file attributes `xattr`, both  get lost during *transportation/downloads*. ... this also applies to `md.kMDItemFinderComment = "Hello World!"` from the https://pypi.org/project/osxmetadata/ examples
* I also looked into the .pkg file content to find a place to store information in an accessible way... this is how the content looks like
```
.
├── Distribution
├── Resources
│   ├── acloud.rtf
│   ├── readme.rtf
│   └── welcome.png
├── cacheclean.pkg
│   ├── Bom
│   ├── PackageInfo
│   ├── Payload
│   └── Scripts
├── pathupdate.pkg
│   ├── Bom
│   ├── PackageInfo
│   ├── Payload
│   └── Scripts
├── prepare_installation.pkg
│   ├── Bom
│   ├── PackageInfo
│   ├── Payload
│   └── Scripts
└── run_installation.pkg
    ├── Bom
    ├── PackageInfo
    ├── Payload
    └── Scripts
```
* Storing something in a rtf was easiest there ... how to store a plain .txt file there or where would any extra plist information pop up there?

Reply:
It would be possible to add the same version comment as in the `header.sh` into the `osx/run_installation.sh` ... but  where does that file end up in above tree to instruct a user to report the value ... 

Reporting the version for the .sh installer is just a `head $installer.sh` und for .exe it is going to the file properties.

Reply:
Figured out the commands to read the run_install.sh from .pkg installers... so using that.